libs/jsapi/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(filamentjs)
```

Comment:
You should call it jsapi to match the directory, or rename the directory to filamentjs

Reply:
Nice catch. Renamed folder to filamentjs.